src/linker/Linker/AssemblyNameHelpers.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+using System.IO;
+using System.Text;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace System.Reflection.Runtime.Assemblies
```

Comment:
The namespace does not match the file path (and tbh it has quite odd name)

Reply:
Should I change all the places where this namespace is used to `Mono.Linker`?

Reply:
No - move the files that come from the runtime to the external directory like Marek suggests below (something like external\TypeParsing\System\Reflection\Runtime\Assemblies) and add a README.md to the root of TypeParsing that tells the source and commit ID that was used as the base. That way we'll have future reference.

Also undo the formatting changes on the files that come from outside so that this is easier to diff, should we have a need.